src/mono/browser/runtime/loader/assets.ts

```diff
@@ -68,6 +68,7 @@ const skipBufferByAssetTypes: {
 } = {
     "dotnetwasm": true,
     "symbols": true,
+    "static-json": true,
```

Comment:
make the asset type more specific, because the instantiation is specific `segmentation-rules` instead of `static-json`